source/maintain_project/review_content/index.html.md.erb

```diff
@@ -0,0 +1,45 @@
+---
+title: Review content regularly
+weight: 30
+last_reviewed_on: 2019-12-04
+review_in: 4 months
+---
+
+# Review content regularly
+
+Make it easier to keep documentation up to date by configuring the [page expiry notifier][daniel-github], Daniel the Manual Spaniel.
+
+## How it works
+
+Daniel the Manual Spaniel scans your documentation pages to see which pages are due for review, according to the `review_by` parameter in your documentation's `/api/pages.json`.
```

Comment:
```suggestion
Daniel the Manual Spaniel checks for pages that are due for review. It looks at the `review_by` dates in your `/api/pages.json` file, which is generated when you build your documentation. 
```

Suggested this rewording to make it a little clearer what the `pages.json` file is.